Apps/Playground/Android/app/src/main/cpp/BabylonNativeJNI.cpp

```diff
@@ -108,6 +109,7 @@ extern "C"
                 Babylon::Polyfills::Window::Initialize(env);
 
                 Babylon::Polyfills::XMLHttpRequest::Initialize(env);
+                Babylon::Polyfills::Canvas::Initialize(env);
```

Comment:
it might be worth having a default font specified during this initialization. Having to do a static call to the native engine to load a font by default feels kind of clunky.

Do we know whether we could distribute any default ttfs with the babylonnative codebase compared to relying on a http request to load one? BabylonNative will need to support offline usage

Reply:
You can bundle ttf fonts in your app (iOS or Android and other) and then load it using app:// scheme

I agree loading a font async when starting the app is far from great.

Reply:
How does this compare to the browser? Do browsers have “built in” fonts, or do you just have access to system fonts, or what? Web pages don’t have to “bundle” a font to be able to display text, including displaying text in Babylon.js correct?

Reply:
Only guess but I think browsers use system font and (maybe) system API to render text. I believe it's possible to render text inside a CPU memory buffer and copy that buffer to the GPU if needed (in the case of Canvas to texture copy). That's something we might evaluate in the future (and compare quality and performance). That said, we can use ttf from the system instead of providing our own on Windows. But it's not possible to have access to fonts on Apple devices.

Reply:
I think I can add a method to do ttf loading when initializing the canvas plugin.

Reply:
Is there an example for ttf loading in a pure typescript context? iirc, providing the ttf data to babylonnative required accessing private/inaccessible babylon.js functions.